app/views/manageBusinesses/manage/ManageIncomeSourceDetails.scala.html

```diff
@@ -160,7 +163,10 @@
                 content = Text(getMessage("reporting-method", (viewModel.latencyDetails.get.taxYear2.toInt - 1).toString, viewModel.latencyDetails.get.taxYear2))
             ),
             value = Value(
-                content = HtmlContent(getMessage(viewModel.latencyValueAsKey(viewModel.latencyDetails.get.latencyIndicator2)))
+                content = if(!viewModel.taxYearTwoCrystallised.get)
+                            HtmlContent(getMessage(viewModel.latencyValueAsKey(viewModel.latencyDetails.get.latencyIndicator2)+".graceperiod"))
+                          else
+                            HtmlContent(getMessage(viewModel.latencyValueAsKey(viewModel.latencyDetails.get.latencyIndicator2)))
```

Comment:
Should FIx: Calling get on option. If the field is mandatory, add the require condition in view model  for fast fail before going to view.

eg. https://github.com/hmrc/income-tax-view-change-frontend/blob/main/app/models/liabilitycalculation/viewmodels/TaxYearSummaryViewModel.scala#L35